CHANGES.md

```diff
@@ -1,7 +1,25 @@
-# next
+# 3.0.0
 
 ## Language changes
 
+* Cryptol now includes a redesigned module system that is significantly more
+  expressive than in previous releases. The new module system includes the
+  following features:
+
+  * Nested modules: Modules may now be defined within other modules.
+
+  * Named interfaces: An interface specifies the parameters to a module.
+    Separating the interface from the parameter declarations makes it possible
+    to have different parameters that use the same interface.
+
+  * Top-level module constraints: These are useful to specify constraints
+    between different module parameters (i.e., ones that come from different
+    interfaces or multiple copies of the same interface).
+
+  See the
+  [manual section]()https://galoisinc.github.io/cryptol/master/Modules.html#instantiation-by-parametrizing-declarations
+  for more information.
+
 * Declarations may now use *numeric constraint guards*.   This is a feature
   that allows a function to behave differently depending on its numeric
   type parameters.  See the [manual section](https://galoisinc.github.io/cryptol/RefMan/_build/html/BasicSyntax.html#numeric-constraint-guards))
```

Comment:
Due to the presence of PR1529, this link (and any other doc links) are already changed (even though that PR isn't merged). Properly I should update PR1529 with these changes, and I can do that unless you just want to handle that here rather than running the merges to get those updates.

Reply:
Good catch. If you could update #1529, I'm happy to wait until that lands so that I can rebase on top of it.

Reply:
Updated #1529.  Turns out there were only two locations, both in this CHANGES.md file.

Reply:
Say, it may be worthwhile to mention the new FFI capability that @qsctr added last summer.

Reply:
I think that's already mentioned here:

https://github.com/GaloisInc/cryptol/blob/4e722ec1ac8294ee4bd60e35a3b7e73472a2968d/CHANGES.md#L10-L12

Or was there something more specific about the FFI that you'd like to see mentioned?

Reply:
Nope! You got it! I missed that somehow.